### PR TITLE
ui: Autocomplete for authors in record editor

### PR DESCRIPTION
**Pull request by hanadikaleel**
#2026

### CODE REVIEW

File: record-editor/src/app/shared/config/authors.config.ts [hunk 1]
```diff
@@ -80,6 +80,9 @@ export const authors: JsonEditorConfig = {
                 anchorBuilder: anchorBuilder,
               },
             },
+            institution: {
+              autocompletionConfig: affiliationAutocompletionConfig,
```
Comment (michamos) [inline record-editor/src/app/shared/config/authors.config.ts:84]: did you test this locally? is it working?

Comment (hanadikaleel) [reply record-editor/src/app/shared/config/authors.config.ts:84]: Yeah, tested and it works

Comment (MJedr) [reply record-editor/src/app/shared/config/authors.config.ts:84]: for me it looks correctly, maybe @oguzdemirbasci can have a look too?

File: record-editor/src/app/shared/config/authors.config.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@
  */
 
 import { JsonEditorConfig } from 'ng2-json-editor';
-import { customValidationForDateTypes, anchorBuilder } from './commons';
+import { affiliationAutocompletionConfig, customValidationForDateTypes, anchorBuilder } from './commons';
```
Comment (michamos) [inline record-editor/src/app/shared/config/authors.config.ts:24]: are you sure you have pre-commit hooks set up correctly? I would expect this to be sorted by `prettier`.

File: record-editor/src/app/shared/config/authors.config.ts (hunk 1 above)
Comment (oguzdemirbasci) [reply record-editor/src/app/shared/config/authors.config.ts:84]: LGTM 👍

File: record-editor/src/app/shared/config/authors.config.ts (hunk 2 above)
Comment (oguzdemirbasci) [reply record-editor/src/app/shared/config/authors.config.ts:24]: Agree with Micha. Line 24 should have been automatically formatted to multiline import statement which is easier to read.

**Pull request closed without merging**